source/how-tos/debug/debug-apache.rst

```diff
@@ -0,0 +1,79 @@
+.. _debug-apache:
+
+How to Debug Apache
+===================
+
+..  tip::
+
+  While this page is helpful it is no replacement for `apache httpd's documentation`. If
+  you are an administrator responsible for Open OnDemand, you are now an administrator of
+  apache httpd.  As such, you should get comfortable with it as from time to time you will
+  have to troubleshoot it.
+
+
+Log locations
+-------------
+
+.. _restart-apache:
+
+Restart services
+----------------
+
+   .. tabs::
+
+      .. tab:: RHEL/CentOS 7
+
+        .. code-block:: sh
+
+          sudo systemctl try-restart httpd24-httpd httpd24-htcacheclean
+
+
+      .. tab:: RHEL/CentOS/Rocky 8
+
+         .. code-block:: sh
+
+          sudo systemctl try-restart httpd htcacheclean
+
+      .. tab:: Ubuntu
+
+         .. code-block:: sh
+
+          sudo systemctl try-restart apache2
+
+.. _show-virtualhosts:
+
+Show VirtualHosts
+-----------------
+
+OpenOnDemand creates it's own VirtualHost in apache.  Apache will route
+requests based off of the hostname in the request to different VirtualHosts.
+
+Showing virtualhosts can help debug apache request routing.  The outpt from these
+commands will show you how apache is routing based off of the servname in the VirtualHost.
+
+If you're seeing the default apache webpage you likely have to `servername configuration`
```

Comment:
Is this sentence missing some words?  Maybe `likely have to configure the ServerName` would read better?

Reply:
Thanks. Yea I was working through some ref labeling issues and must have left this.